module/users/helper.js

```diff
@@ -163,4 +163,78 @@ module.exports = class UsersHelper {
             }
         });
     }
+
+
+    /**
+   * Search user.
+   * @method
+   * @name search
+   * @param {String} token - user access token
+   * @param {String} userId - user id
+   * @param {String} organisationId - organisation id
+   * @param {String} pageSize - maximum limit 
+   * @param {String} pageNo - page number
+   * @param {String} searchText - search text of users
+   * @param {String} status - status of the users
+   * @param {Array} requestedUsers - array of selected user id
+   * @param {String} email - user email address
+   * @param {String} phone - user phone number
+   * @returns {json} Response consists of users of organisation.
+   */
+
+    static search(token, userId, userName, pageSize, pageNo, searchText, status = "", requestedUsers = [],email="",phone="") {
```

Comment:
@rakeshSgr - Can we just name it searchByUsername here and in API so its clear no other query will come.

Reply:
It's generic code, we can search by name, email and  using other fields as well